integration/hsm/hsm_test.go

```diff
@@ -555,10 +554,15 @@ func TestHSMRevert(t *testing.T) {
 	require.NoError(t, err)
 
 	// Make sure a cluster alert is created.
-	alerts, err := auth1.process.GetAuthServer().GetClusterAlerts(ctx, types.GetClusterAlertsRequest{})
-	require.NoError(t, err)
-	require.Len(t, alerts, 1)
-	alert := alerts[0]
+	var alert types.ClusterAlert
+	require.EventuallyWithT(t, func(t *assert.CollectT) {
+		// Keep advancing the clock to make sure the rotation ticker gets fired
+		clock.Advance(2 * defaults.HighResPollingPeriod)
```

Comment:
I really don't like this style of test as it's still not deterministic, we're just hoping that if we advance the clock enough the right thing happens.

Would a synctest test help here?

Reply:
I see.